docs/index.html

```diff
@@ -181,14 +183,10 @@
 evaluates to the `integer` `3`.  The `{}` block ensures that `one` and `two`
 are not accessible after `three` is computed.
 
-```reason
-let three = {
-  let one = 1;
-  let two = 2;
-  one + two
-};
-/* one and two not accessible here! */
-```
+<pre>
+<p data-id="-P8x86Mg6w" data-host="replay-154206.appspot.com" data-height="256" data-font-size="12px" class="replay" ></p>
```

Comment:
Not sure how often you may change the code snippet, but if you just want to see the code content directly in the html file, we can add the code in the <p> tag replay generated like this:
```
<p data-id="-P8x86Mg6w" data-host="replay-154206.appspot.com" data-height="256" data-font-size="12px" class="replay" >
<code>
let three = {
  let one = 1;
  let two = 2;
  one + two
};
print_int three
</code>
</p>
```
but changing the code here will not change the code in output embedded content